api/controllers/insert.js

```diff
@@ -9,7 +9,7 @@ module.exports = function(store) {
 	return [
 		requireUserAdmin(),
 		ensureFirstLastname(),
-		validateBodyRoleAssignments(),
+		// validateBodyRoleAssignments(),
```

Comment:
Be **very careful when commenting out middleware inclusions**. Was it just for testing purposes, or did you consider it obsolete? I have added a test that demonstrates a severe security breach when posting users administratively. Please update and run the tests.